app/manifest.json

```diff
@@ -1,36 +1,37 @@
 {
     "name": "__MSG_appName__",
-    "version": "0.10.0",
+    "version": "1.0.2",
     "manifest_version": 2,
     "description": "__MSG_appDescription__",
     "icons": {
         "16": "images/icon-16.png",
         "128": "images/icon-128.png"
     },
+    "applications": {
+        "gecko": {
+            "id": "addon@fullyfeedly"
+        }
+    },
     "default_locale": "en",
     "background": {
-        "scripts": [
-            "scripts/chromereload.js",
-            "scripts/background.js"
-        ],
-        "persistent": false
+        "scripts": ["scripts/chromereload.js", "scripts/browser-polyfill.js", "scripts/background.js"]
```

Comment:
Is polyfill used to provide backward compatibility to certain browsers?

Reply:
Yes. The polyfill is used to provide compatibility for browsers that implement the chrome.* APIs but not the replacement browser.* APIs (i.e. Chromium-based browsers). Browser.* support is coming in Chromium, but it's not available yet, so this enables the same code to work across browsers.